java/client/src/main/java/glide/api/models/commands/RangeOptions.java

```diff
@@ -15,6 +16,7 @@
  *
  * @see <a href="https://redis.io/commands/zrange/">redis.io</a>
  */
+@Builder
```

Comment:
Why make this a builder now?

Reply:
Why not? Seems like most of our options are builders?

Reply:
If it's just one or two required arguments, builders are not really necessary.

Reply:
The `@RequiredArgsConstructor` replaces the need for a Builder, IMO

Reply:
It doesn't. When you have the required args constructor you will have to pass both arguments. But both are not required together. I will have to create my own constructors as Yury suggested. But why not stick with consistency and use a builder if most of our options uses a builder?